src/utils/dataUtils.js

```
@@ -0,0 +1,213 @@
+      const bValue = b.data[sort.fieldId];
+
+      let comparison = 0;
+
+      if (field.type === FIELD_TYPES.NUMBER) {
+        comparison = (Number(aValue) || 0) - (Number(bValue) || 0);
+      } else if (field.type === FIELD_TYPES.DATE) {
+        const aDate = aValue ? new Date(aValue) : new Date(0);
+        const bDate = bValue ? new Date(bValue) : new Date(0);
+        comparison = aDate.getTime() - bDate.getTime();
+      } else if (field.type === FIELD_TYPES.CHECKBOX) {
+        comparison = (Boolean(bValue) ? 1 : 0) - (Boolean(aValue) ? 1 : 0);
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)